docs/_docs/admin-guide/admin-installation.md

```diff
@@ -48,7 +48,39 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
 Alternatively, you can edit the `CODENVY_HOST` value in `codenvy.env`. 
 
 ## Ports
-TODO - UPDATE PORT DESCRIPTION SECTION
+The master node is where Codenvy is installed and running. In a [scalability mode]({{base}}/docs/admin-guide/managing/index.html), you can add additional physical "workspace" nodes to increase system capacity. If you have not added any additional physical workspace nodes, then the Codenvy master node doubles as a workspace node and needs both sets of ports opened.
+
+#### Master Node: External
+|Port|Service|Notes|
+|---|---|---|
+|80 / 443|HAProxy HTTP/S|HTTP is the default. If you configure [HTTP/S]({{base}}/docs/admin-guide/configuration/index.html#https), then port 80 can be closed.
+|5000|Docker Registry|Embedded registry to save workspace snapshots. This port is not required if you configure an external registry.
```

Comment:
As a simplification of the documentation, we are going to treat the system as if there is only one configuration - AIO+.  So it does not hurt to show additional ports if you are only running AIO.  It'll be simpler to maintain.

But doesn't 5000 need to be open on both master and workspace nodes so that they can communicate back and forth?